doc/source/introduction_intro.rst

```diff
@@ -14,49 +14,93 @@ to facilitate access to and exchange of genomics data across remote sites.
 Why use this web API?
 --------------------------
 
-This API is specifically designed to allow sharing of genomics data without having to exchange complete experiments.
-With this API, you can
+You can use this API to share genomics data without needing to copy files around or worry about details of file formats:
 
 * Exchange genome annotations, DNA sequence reads, reference-based alignments, metadata, and variant calls
-* Request alignments and variant calls for one genome or a million.
+* Request alignments and variant calls for one genome, or a million
 * Explore data by slicing alignments and variants by genomic range across one or multiple samples
 * Interactively process entire cohorts
 
 --------------------------
-If you need background
+The GA4GH web API
 --------------------------
-For more details on web APIs, see `this wikipedia page <https://en.wikipedia.org/wiki/Web_API>`_  
 
-ReST protocols for data transfer are described `here <https://en.wikipedia.org/wiki/Representational_state_transfer>`_
+This API was created to enable researchers to better access and exchange genomic data across remote sites. Instead of having to
+download large BAM files or whole genome annotations, the API allows you to retrieve data about, for instance, single genes or genomic
+regions.  The API is designed for a variety of uses, from interactive visualization to large-scale analysis.
+
+For a full list of the GA4GH API goals, see :ref:`apigoals`
+
+The API consists of a series of :ref:`apidefinition`, or schemas, that define datasets, metadata, read group sets, reads, variants, etc. 
+
+The GA4GH web API schemas show developers how to connect data consumers ('clients') to data providers ('servers').
+They define how the data is organized, and thereby give information on what can be requested.
+The schemas are defined in Avro Interactive Data Language (extension .avdl), and 
+can be used to create code in any programming language.
```

Comment:
The Avro project site calls Avro a "data serialization system."  Below you term it a "data format definition language." Either wording would be better than "Interactive Data Language."